pkg/lib/machine-change-auth.html

```diff
@@ -10,7 +10,11 @@ <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
     {{/available}}
 
     {{#available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}. You can change your authentication credentials below. {{#can_sync}}You may prefer to {{#sync_link}}synchronize accounts and passwords{{/sync_link}}.{{/can_sync}}</p>
+    <p translate="yes">Cockpit was unable to log into
```

Comment:
Should scrub project name from this, to make it just "was unable to log..."
Also needs space between in and to, so it's "in to", rather than "into"

Reply:
Done.